Dockerfile

```diff
@@ -1,26 +1,17 @@
-# To build locally: docker buildx build . -t wakapi --load
-
-# Preparation to save some time
-FROM --platform=$BUILDPLATFORM golang:1.18-alpine AS prep-env
```

Comment:
Do you know that this `--platform` tag was used for?

Reply:
To run the `prep-env` stage only once (on the host platform) when building for multiple targets for release, from #293 . 
However, it seems that `go build` still downloads packages. As this is only used during building releases, I feel that the minor performance enhancement (if any exists) is not worthwhile when compared to readability and ease of building with the regular `docker build`.